csrc/scheduler/mma_utils.cpp

```diff
@@ -466,18 +467,18 @@ void orderTiledConcreteIdAsRoot(TensorView* tv) {
       leftmost_pos = i;
       continue;
     }
-    auto maybe_root =
-        getMaybeRootIfInnermostTiled(leaf_id, maybe_rfactor_id_set);
+    auto maybe_alloc_domain =
+        getMaybeAllocationIfInnermostTiled(leaf_id, id_set);
```

Comment:
nitpick: I think the logic here assumes rfactor and allocation are just permutation? Should we add an assert somewhere?

Reply:
So I guess if we are doing that somewhere here (or maybe it's already done in an even earlier spot, please ignore the comment on the other place where I asked to assert the permutation.